### ISSUE TITLE
I can not build template project for Nucleo-H745ZI-Q board.

### ISSUE THREAD

**Issue by Catutigernos** [libopencm3/libopencm3#1533]
Hi everyone,
I try to your template project for my Nucleo H745ZI-Q board. I try to set DEVICE stm32h7, stm32h7x, stm32h745zit6u etc. every time, return an error, because it could not find the device in the "devices.data" file. What should I do to build project?

Thanks in advance.

**Comment by Catutigernos** [comment]
Is project active?

**Comment by karlp** [comment]
yes and no.  devices.data file doesn't have every part ever made (yet?) 
You can add patterns to https://github.com/libopencm3/libopencm3/blob/master/ld/devices.data#L265-L275 locally to match your parts if you like.
